angr/knowledge_plugins/cfg/cfg_node.py

```diff
@@ -131,14 +131,32 @@ def successors(self):
     def predecessors(self):
         return self._cfg_model.get_predecessors(self)
 
+    def get_data_references(self, kb=None):
+        if self._cfg_model.ident != 'CFGFast':
+            raise ValueError("Memory data is currently only supported in CFGFast.")
+        if not kb:
+            kb = self._cfg_model.project.kb
+        if not kb:
+            raise ValueError("The Knowledge Base does not exist!")
+
+        for instr_addr in self.instruction_addrs:
+            refs = list(kb.xrefs.get_xrefs_by_ins_addr(instr_addr))
+            for ref in refs:
+                yield ref
+
+
     @property
     def accessed_data_references(self):
```

Comment:
Can we make `accessed_data_references` to use the new `get_data_references()` method to avoid code duplication?